### PR TITLE
Fixing the startup script to user the correct folder

### PR DESCRIPTION
**Pull request by digimatspa**
PIDDIR=/var/run/postgresql   ->  PIDDIR=/var/run/pgbouncer

This change permits to restart the service without errors

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@
     state: "{{ pgbouncer_install_state }}"
     update_cache: yes
     disable_gpg_check: yes
+    disablerepo: 'epel'
```
Comment (corrado9999) [inline tasks/main.yml:13]: How does this relate to the startup script issue?

**Review by corrado9999 — APPROVED** [review]

**Comment by corrado9999** [comment]
@digimatspa I do not remember, can I merge this or do you have further commits?

**Comment by digimatspa** [comment]
@corrado9999 
I have a couple of other things to commit: reload service and restart service.
If you prefer another merge request let me know  . Otherwise I can also commit here

**Comment by corrado9999** [comment]
Please, commit here.

**Comment by digimatspa** [comment]
I'll add another change here for CentOS 6 since it's related the previous commits
@corrado9999

**Review by corrado9999 — APPROVED** [review]

**Pull request merged**